torch/testing/_internal/schema_check_mode.py

```diff
@@ -23,6 +24,21 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
         def has_mutated(before, after):
             return not torch.equal(before, after) if isinstance(before, torch.Tensor) and isinstance(after, torch.Tensor) else False
 
+        def have_values_aliased(lhs, rhs):
+            return torch._C._is_alias_of(lhs, rhs) if isinstance(lhs, torch.Tensor) and isinstance(rhs, torch.Tensor) else False
+
+        def is_aliasing(lhs_argument, rhs_argument):
+            return bool(len(lhs_argument.before_set & rhs_argument.before_set))
+
+        def unwrap(e):
+            if isinstance(e, torch.Tensor) and not type(e) == torch.Tensor:
```

Comment:
nit: could you add a comment about why you need this? (e.g. to handle IncorrectAliasTensor)